src/components/Restoration/RestorationType.vue

```diff
@@ -39,15 +43,20 @@
               </div>
             </v-expand-transition>
 
-            <v-radio id="limited-radio-button" label="Limited Restoration" :value=RestorationTypes.LIMITED />
+            <v-radio
+              class="pt-4 radio-button"
+              id="limited-radio-button"
+              label="Limited Restoration"
+              :value=RestorationTypes.LIMITED />
 
             <!-- Limited Restoration Radio Panel -->
             <v-expand-transition>
-              <div v-if="isLimitedRestorationFiling">
+              <div v-if="isLimitedRestorationFiling" class="mb-n12">
```

Comment:
Decrease the spacing between the restoration type and approval type.

Reply:
In my opinion, you should only use negative bottom margin if this component is too tall... and there's usually a better way to fix that.

The spacing between component should be set in the parent. THIS component should not decrease the spacing to the next component.

Reply:
I was actually debating whether I should fix this spacing in the parent or in this component. I got my answer. Thanks Sev.

Reply:
Components should have "tight" spacing (no whitespace around them) so they can be reused more easily.